account_invoice_report_grouped_by_picking_optional_journal/models/account_journal.py

```diff
@@ -0,0 +1,10 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    show_group_by_picking_report = fields.Boolean(default=False, string="Show Group by Picking")
```

Comment:
`default` value should be the opposite (`False` actually will disable feature, and this addon, once installed, should keep previous behavior), improve field label, because "Show Group By Picking" definition misses key word "report" and a more detailed `help`:
```suggestion
    show_group_by_picking_report = fields.Boolean(
        default=True,
        string="Invoice Report - Show Grouped by Picking",
        help="Leave unmarked if original Odoo behavior (not grouped) is required for invoices belonging this journal",
    )
```